_run_app_dependencies.sh

```diff
@@ -105,6 +105,19 @@ if confirm "Run pnpm run build?"; then
 	pnpm run build
 else
 	echo "Skipping pnpm run build"
+	for i in $(seq 30 -1 1); do
```

Comment:
remove this ;) instead try to use the wait for script that we already apply in cypress workflow and have in util